components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java

```diff
@@ -1709,9 +1709,11 @@ public ServiceProvider getApplication(String applicationName, String tenantDomai
             if (serviceProvider != null) {
                 loadApplicationPermissions(applicationName, serviceProvider);
             }
+            connection.commit();
```

Comment:
we need to use same style.
IdentityDatabaseUtil.commit(connection) 
as we use IdentityDatabaseUtil for getting and rollback of the connection